tests/templates/kuttl/kerberos/12-rego-rules.yaml.j2

```diff
@@ -0,0 +1,234 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: hdfs-regorules
+        labels:
+          opa.stackable.tech/bundle: "true"
+      data:
+        hdfs.rego: |
+          package hdfs
+
+          import rego.v1
+
+          default allow = false
+
+          # HDFS authorizer
+          allow if {
+              some acl in acls
+              matches_identity(acl.identity)
+              matches_resource(input.path, acl.resource)
+              action_sufficient_for_operation(acl.action, input.operationName)
+          }
+
+          # Identity mentions the (long) userName explicitly
+          matches_identity(identity) if {
```

Comment:
maybe mention that the three `matches_identity` calls are OR-ed together?

Reply:
Hmm I personally would argue that this is basic rego rule knowledge and would spoil the rules but not entirely sure what is better. The group in Karlsruhe decided to leave it as is and had an improvement in dba095f